tests/conftest.py

```diff
@@ -167,36 +195,66 @@ def pytest_addoption(parser):
     group.addoption(
         "--integration",
         action="append",
+        dest="integration",
         metavar="INTEGRATIONS",
         help="only run tests matching integration specified: "
         "[cassandra,kerberos,mongo,celery,statsd,trino]. ",
     )
+    group.addoption(
+        "--skip-db-tests",
+        action="store_true",
+        dest="skip_db_tests",
+        help="skip tests that require database",
+    )
+    group.addoption(
+        "--run-db-tests-only",
+        action="store_true",
+        dest="run_db_tests_only",
+        help="only run tests requiring database",
+    )
     group.addoption(
         "--backend",
         action="store",
+        dest="backend",
         metavar="BACKEND",
         help="only run tests matching the backend: [sqlite,postgres,mysql].",
     )
     group.addoption(
         "--system",
         action="append",
+        dest="system",
         metavar="SYSTEMS",
         help="only run tests matching the system specified [google.cloud, google.marketing_platform]",
     )
     group.addoption(
         "--include-long-running",
         action="store_true",
+        dest="include_long_running",
         help="Includes long running tests (marked with long_running marker). They are skipped by default.",
     )
     group.addoption(
         "--include-quarantined",
         action="store_true",
+        dest="include_quarantined",
         help="Includes quarantined tests (marked with quarantined marker). They are skipped by default.",
     )
+    group.addoption(
```

Comment:
By default Virtualenv tests are included - which makes them easy to test in IDE, but in CI, since they are long running and have potential for contention/stalling, we separated them out to much smaller test types, which allows to parallelise them much better. They are excluded from `Operators` test type by those markers.